src/app/giftapp/model/product.ts

```diff
@@ -0,0 +1,17 @@
+export class Product {
```

Comment:
zamiast tego proponuję zastosować interfejs, prościej, czytelniej.

Reply:
Chodziło mi o to, żeby można było konstruować Product bez pamiętania jak się nazywają poszczególne pola - za pomocą konstruktora. Chyba zostawię jako klasę

Reply:
Można, tak też jest ok